### PR TITLE
LSNBLDR-961 Bypass store chooser screen if there is only one.

### PR DESCRIPTION
**Pull request by csev**

### CODE REVIEW

**Comment by csev** [comment]
This builds on LSNBLDR-929

**Comment by csev** [comment]
For some reason I cannot include @davidpbauer as a reviewer :(

**Comment by csev** [comment]
Matt / Neal - Or anyone in the depths of core - should I fix this to be LSNBLDR?  If so - how do I go about doing this?

File: lessonbuilder/tool/src/webapp/templates/BltiPicker.html [hunk 1]
```diff
@@ -119,6 +131,21 @@ <h2 rsf:id="mainhead" id="mainhead"></h2>
 		console.log(e.data);
 	}
 });
+
+// See if we are supposed to launch the shortcut link
+var shortcut_tag = document.getElementById('app-store-shortcut::blti-create::blti-create-link');
+console.log(shortcut_tag);
```
Comment (ottenhoff) [inline lessonbuilder/tool/src/webapp/templates/BltiPicker.html]: please remove console.log

Comment (csev) [reply lessonbuilder/tool/src/webapp/templates/BltiPicker.html]: Do I need to rename my branch or just update the original SAK- branch.

Comment (ottenhoff) [reply lessonbuilder/tool/src/webapp/templates/BltiPicker.html]: No need to change your branch. just add more commits and push and they will be auto-updated here.

**Comment by csev** [comment]
I think this should just go into master.

**Comment by csev** [comment]
Thanks.

**Pull request merged**